docs/user_guide/network.rst

```diff
@@ -330,4 +330,49 @@ You can now run a loadflow to check our network actually works !
     'ComponentStatus.CONVERGED'
 
 For more details and examples about network elements creations,
-please refer to the API reference :doc:`documentation </reference/network>`.
\ No newline at end of file
+please refer to the API reference :doc:`documentation </reference/network>`.
+
+
+Modify an existing network
+--------------------------
+
+If you have a network in node/breaker topology, you can use pypowsybl to add injections with the associated topology.
+Let's take an example. First you can create a node/breaker simple network with:
+
+.. testcode::
+
+    network = pp.network.create_four_substations_node_breaker_network()
+
+You can display the voltage level 'S1VL2' with a single line diagram:
+
+.. testcode::
+
+    network.get_single_line_diagram('S1VL2')
+
+.. image:: ../_static/images/four_substation_node_breaker_s1vl2.svg
+
+Then you need to create a dataframe with the attributes of the injection that you want to add to the network. Let's say
+that we want to add a load, with the id "new_load" on the voltage level S1VL2. The load will have a p0 of 100 and a
+q0 of 50. It is also necessary to put in the dataframe the id of the busbar section where we want to put the load and
+the order position.
+
+.. testcode::
+
+    df = pd.DataFrame(index=["new_load"], columns=["id", "p0", "q0", "voltage_level_id", "busbar_section_id", "position_order"], data=[["new_load", 100.0, 50.0, "S1VL2", "S1VL2_BBS1", 10]])
```

Comment:
I think the documentation needs to evolve a lot in the future. Either when other hypothesis are available, to create the network using all of them or at least when the extensions will be available, to show really all the possibilities offered. I did not put too much for now because I really think there will be a lot to evolve.

Reply:
Ok!